inferno-framework/inferno-core
# feature/FI-1645-options-redesign

# Summary
Redesigns landing and suite options pages for visual appeal and reactivity.

<img width="600" alt="Screen Shot 2022-08-12 at 3 47 51 PM" src="https://user-images.githubusercontent.com/11209247/184432305-a86b6804-320f-4090-baf6-49d4a90e5f63.png">
<img width="600" alt="Screen Shot 2022-08-12 at 3 47 57 PM" src="https://user-images.githubusercontent.com/11209247/184432321-109a81d8-5b95-406b-9a4b-80c232053498.png">

# Testing Guidance
- Confirm that selecting suites and options works as normal
- Confirm that window responsiveness works (e.g. resize windows)
- Check that arrow button on Suite Options Page returns back to the Landing Page

# Notes
Open to feedback on design!
